source/_integrations/plum_lightpad.markdown

```diff
@@ -20,7 +22,8 @@ Configurable/Dimmable Wi-Fi Lightswitch
 
 ## Configuration
 
-To enable Plum Lightpad support, add the following lines to your `configuration.yaml` file:
+To use Plum Lightpad devices in your installation, add your Plum account from the integrations page.
+Alternatively, add the following lines to your `configuration.yaml` file:
```

Comment:
Isn't the config deprecated? If so, we should remove it.

Reply:
Hi Frenk. The config is still being supported by this integration, so adding config flow is transparent to pre-existing users. Do you want me to keep this reference in this case? Or is it the best practice to remove support from being documented?

Reply:
Normally we deprecate YAML config if it was fully replaced with frontend.
But if it isn't deprecated, then it is fine to leave it in.